app/assets/javascripts/discourse/app/components/post-list/item/details.gjs

```diff
@@ -1,65 +1,95 @@
 import Component from "@glimmer/component";
+import { tracked } from "@glimmer/tracking";
 import { hash } from "@ember/helper";
 import { htmlSafe } from "@ember/template";
 import PluginOutlet from "discourse/components/plugin-outlet";
+import TopicStatus from "discourse/components/topic-status";
 import categoryLink from "discourse/helpers/category-link";
 import getURL from "discourse/lib/get-url";
 import { prioritizeNameInUx } from "discourse/lib/settings";
 import { i18n } from "discourse-i18n";
 
 export default class PostListItemDetails extends Component {
-  get titleAriaLabel() {
+  @tracked
+  url =
+    this.args.post?.url || this.args.post?.post_url || this.args.post?.postUrl;
+
+  get showUserInfo() {
+    if (this.args.showUserInfo !== undefined) {
+      return this.args.showUserInfo && this.args.user;
+    }
+
+    return this.args.user;
+  }
+
+  get topicTitle() {
     return (
-      this.args.titleAriaLabel ||
-      i18n("post_list.aria_post_number", {
-        title: this.args.post.title,
-        postNumber: this.args.post.post_number,
-      })
+      this.args.post?.topic_html_title ||
+      this.args.post?.topic_title ||
+      this.args.post?.title
```

Comment:
This is... weird? I guess we're having to cope with the result of different APIs?

Perhaps one of the arguments to `PostList` should be `titlePath`? We use that pattern [for the avatar helper](https://github.com/discourse/discourse/blob/cf0a4a7e119498b3db045807ad4ee247cdd3fea4/app/assets/javascripts/discourse/app/helpers/avatar.js#L39C40-L39C52)